index.html

```diff
@@ -155,10 +177,11 @@ <h1 class="post">Multi-Post Stories</h1>
           </li>
           <li>Ruby</li>
         </ul>
-        <div class="but">
+        <div class="but2">
           <button class="bt" type="button">see project</button>
         </div>
       </section>
+      <!--Middle3-->
       <section class="middle3">
         <img
```

Comment:
Nice implementing the design, however, you should add some padding with an 804px widescreen so the left side will be the same as the right side 
![image](https://user-images.githubusercontent.com/61790587/165863851-50fc6c94-cddf-4be3-a8a5-53ef7aade05b.png)